docs/markdown/Introduction/media.md

```diff
@@ -200,6 +200,50 @@ Sept 3, 2020
 
 ## Talks
 
+### PyCon AU 2023
+
+#### Packaging for serverless: effortless? Doubtless!
+August 19, 2023
+<https://www.youtube.com/watch?v=YwuUI6bYUh0&t=5s&pp=ygUwUGFja2FnaW5nIGZvciBzZXJ2ZXJsZXNzOiBlZmZvcnRsZXNzPyBEb3VidGxlc3Mh>
+
+> Pants team member Huon Wilson talks about how the Pants build system works in practice for getting our Python code running in AWS Lambdas in production, and how it's improved upon other common practices we previously used.
+
+### Pycon China 2022
+
+#### [LIGHTNING TALK] 沈达：Pants，Python工程化必备构建工具
+December 17, 2022
+<https://www.bilibili.com/video/BV1L3411S76J/>
+
+> 这是一次闪电演讲，在此次演讲中，Darcy Shen简明扼要地介绍了他使用Pants做Python项目工程化的经历。
+
+### Tubi China
+
+#### Pants : Python 工程化的必备工具
+
+December 1, 2022
+<https://www.bilibili.com/video/BV1L3411S76J/>
+
+> In this internal tech talk, TubiTV data engineer Darcy Shen illustrates: one command line to launch JupyterLab with a proper dependency set, writing Python snippets just using a web browser, best practices for managing Python projects with Pants, and smart dependency inference of Pants.
+
+### Pycon Japan
+
+#### Modernizing development workflow for a 7-year old 74K LoC Python project using Pantsbuild
+
+October 15, 2022
+<https://www.youtube.com/watch?v=nZr1Wks07vc&t=3459s>
+
+> Mono-repository or not? That is a boggling question for many medium-to-large-sized development teams. As a growing company, Lablup had to onboard new hires quickly while coping with flooding customer requests and increasing codebase complexity. They merged 7 repositories into a single one and migrated to the Pantsbuild system, a Python-friendly modern build system. Here is the story, as told by CTO Joongi Kim.
```

Comment:
Hello @achimnol, I think this is referring to a talk you did. The YouTube link seems like it doesn't work any more. Do you have a better link for more info about this talk?

Thanks!